server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -662,18 +672,50 @@ const orderMutations = {
         if (!req.remoteUser.isAdminOfCollective(fromAccount)) {
           throw new Unauthorized();
         }
-      }
 
-      const toAccount = await fetchAccountWithReference(paymentIntentInput.toAccount, { throwIfMissing: true });
-      const hostStripeAccount = await toAccount.getHostStripeAccount();
-      const host = await toAccount.getHostCollective();
+        let pm = await models.PaymentMethod.findOne({
+          where: {
+            CollectiveId: fromAccount.id,
+            service: PAYMENT_METHOD_SERVICE.STRIPE,
+            type: PAYMENT_METHOD_TYPE.PAYMENT_INTENT,
+            data: {
+              stripeAccount: hostStripeAccount.username,
+            },
+          },
+        });
+
+        const customer = await stripe.customers.create(
+          {
+            email: req.remoteUser.email,
+            description: `${config.host.website}/${fromAccount.slug}`,
+          },
+          !isPlatformHost
+            ? {
+                stripeAccount: hostStripeAccount.username,
+              }
+            : undefined,
+        );
+
+        if (!pm) {
+          pm = await models.PaymentMethod.create({
+            customerId: customer.id,
+            CreatedByUserId: req.remoteUser.id,
+            CollectiveId: fromAccount.id,
+            service: PAYMENT_METHOD_SERVICE.STRIPE,
+            type: PAYMENT_METHOD_TYPE.PAYMENT_INTENT,
```

Comment:
If we're going to set SEPA and ACH payment method types, I think we should deprecate the generic `PAYMENT_INTENT`.

Reply:
For payments done through the stripe payment element we dont know what payment method was attached to it until after confirmation. In this scenario we still need some generic payment method, at least until we received the webhook. this is also saving the stripe customer id for the host account. wdyt?

Reply:
In that case, we should update this existing payment method when the information is attached.

Reply:
Done, we attach a proper payment method when the payment intent is processed.